app.js

```diff
@@ -120,6 +98,23 @@ function parseCommand(command) {
   return parsed;
 }
 
+app.command(slashCommand, async ({ command, ack, respond }) => {
+  await ack();
+  const userCommand = parseCommand(command);
+
+  switch (userCommand.command) {
+    case "help":
```

Comment:
Might wanna add a `default` case reporting "Unsupported command" or at least logging the fact that an unsupported command was used.

Reply:
done